server/store/crypt_test.go

```diff
@@ -57,3 +57,72 @@ func TestEncrypt(t *testing.T) {
 		})
 	}
 }
+
+func TestEncrypt(t *testing.T) {
+	for _, test := range []struct {
+		Name          string
+		Key           []byte
+		ExpectedError string
+	}{
+		{
+			Name:          "Encrypt: Invalid key",
+			Key:           make([]byte, 1),
+			ExpectedError: "could not create a cipher block",
+		},
+		{
+			Name: "Encrypt: Valid",
+			Key:  make([]byte, 16),
+		},
+	} {
+		t.Run(test.Name, func(t *testing.T) {
+			assert := assert.New(t)
+			resp, err := encrypt(test.Key, "mockData")
+			if test.ExpectedError != "" {
+				assert.Contains(err.Error(), test.ExpectedError)
+				assert.Equal("", resp)
+			} else {
+				assert.Nil(err)
+				assert.NotEqual("", resp)
```

Comment:
Here is one of my main concerns. This test is saying anything that the `encrypt` function returns that is not empty, is a valid result for encrypting "mockData" string. so if encrypts always returns "fake", it will pass. If I change the implementation of encrypt, as far as it return a non empty string, it will pass. The same for the decrypt method. Returning not empty string is not a valid enough check.

Reply:
ok, that makes sense.